### PR TITLE
initial react test

### PR DESCRIPTION
**Pull request by cgirod3**

### CODE REVIEW

File: templates/jason.html [hunk 1]
```diff
@@ -1 +1,35 @@
-<h3> HIIIII </h3>
```
Comment (peterdolan) [inline templates/jason.html:1]: nit: leave this

File: templates/jason.html [hunk 2]
```diff
@@ -1 +1,35 @@
...
+      <br>
+      <div id="content"></div>
+    </div>
+    <!-- scripts -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.1.0/react.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.1.0/react-dom.min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/react/0.13.3/JSXTransformer.js"></script>
+    <script type="text/jsx">
+
+      /*** @jsx React.DOM */
```
Comment (peterdolan) [inline templates/jason.html:20]: nit: remove comments

File: templates/jason.html [hunk 3]
```diff
@@ -1 +1,35 @@
...
+      });
+
+      ReactDOM.render(
+        React.createElement(realPython, null),
+        document.getElementById('content')
+      );
+
+    </script>
+  </body>
+</html>
```
Comment (peterdolan) [inline templates/jason.html]: nit: no newline

**Review by peterdolan — CHANGES_REQUESTED** [review]
Needs some work -- please do a more thorough read through next time

File: templates/jason.html (hunk 1 above)
Comment (cgirod3) [reply templates/jason.html:1]: bad code. removed

File: templates/jason.html (hunk 2 above)
Comment (cgirod3) [reply templates/jason.html:20]: no.

File: templates/jason.html (hunk 3 above)
Comment (cgirod3) [reply templates/jason.html]: fixed

File: templates/jason.html (hunk 1 above)
Comment (peterdolan) [reply templates/jason.html:1]: Let's take this discussion offline

**Review by peterdolan — APPROVED** [review]
LOTM

o = okay

**Pull request merged**